src/controllers/DynamicFacetQueryController.ts

```diff
@@ -1,18 +1,18 @@
-import { DynamicFacet } from '../ui/DynamicFacet/DynamicFacet';
 import { QueryBuilder } from '../ui/Base/QueryBuilder';
 import { Assert } from '../misc/Assert';
 import { IFacetRequest, IFacetRequestValue } from '../rest/Facet/FacetRequest';
 import { FacetSortCriteria } from '../rest/Facet/FacetSortCriteria';
 import { QueryEvents } from '../events/QueryEvents';
 import { findIndex } from 'underscore';
 import { IQueryResults } from '../rest/QueryResults';
+import { IDynamicFacet } from '../ui/DynamicFacet/DynamicFacetInterface';
```

Comment:
For the file name, I prefer to use a capital I at the start rather than the word interface. I think this removes any ambiguity with a visual user interface e.g. `SearchInterface.ts`, and is consistent with other interface files e.g. `IFacetSearch`